scripts/bulk-upload-documents.py

```diff
@@ -0,0 +1,40 @@
+"""
+This will scan a directory for files with a filename matching -<supplier_id>_Framework_Agreement
```

Comment:
The description and example usage needs to be updated to explain the new <document_type> parameter